core/client/hammer/hammer.go

```diff
@@ -19,15 +19,16 @@ import (
 	"context"
 	"fmt"
 	"log"
-	"sync"
 	"time"
 
-	"github.com/golang/glog"
 	"github.com/google/tink/go/tink"
+	"golang.org/x/time/rate"
 	"google.golang.org/grpc"
 
 	"github.com/google/keytransparency/core/client"
+	"github.com/google/keytransparency/core/mutator/entry"
 
+	cpb "github.com/cheggaaa/pb"
```

Comment:
I don't think this is the canonical import path for this new dependency - should it be `gopkg.in/cheggaaa/pb.v1` instead?

Reply:
I've removed the progress bar and this import.